app/app.go

```diff
@@ -450,6 +458,45 @@ func wireCoreWorkflow(ctx context.Context, life *lifecycle.Manager, conf Config,
 	return nil
 }
 
+// wirePrioritise wires the priority protocol which determines cluster wide priorities for the next epoch.
+func wirePrioritise(conf Config, life *lifecycle.Manager, tcpNode host.Host, peers []peer.ID,
+	sendFunc p2p.SendReceiveFunc, coreCons core.Consensus, sched core.Scheduler, p2pKey *ecdsa.PrivateKey,
+) error {
+	cons, ok := coreCons.(*consensus.Component)
+	if !ok {
+		// Priority protocol not supported for leader cast.
+		return nil
+	}
+
+	// consensusDelay of 6 seconds is a good start. It is long enough for all peer to share proposals both in prod and in testing.
+	const consensusDelay = time.Second * 6
```

Comment:
is the delay time (6s) related to half-time of a slot?

Reply:
yeah